site/no-nb/index.html

```diff
@@ -0,0 +1,669 @@
+  MutableList&lt;String&gt; expectedFirstNames =
+    Lists.mutable.with("Mary", "Bob", "Ted", "Jake", "Barry", "Terry", "Harry", "John");
+  Assert.assertEquals(expectedFirstNames, firstNames);
+}</pre>
+                        </div>
+                        <div class="tab-pane fade" id="solution">
+                            <pre  class="prettyprint lang-java">
+@Test
+public void getFirstNamesOfAllPeople()
+{
+  MutableList&lt;Person&gt; people = this.people;
+  MutableList&lt;String&gt; firstNames = people.collect(Person::getFirstName);
+  MutableList&lt;String&gt; expectedFirstNames =
+    Lists.mutable.with("Mary", "Bob", "Ted", "Jake", "Barry", "Terry", "Harry", "John");
+  Assert.assertEquals(expectedFirstNames, firstNames);
+}</pre>
+                        </div>
+                    </div>
+                    <!-- /.tab-content -->
+                </div>
+                <!-- /.col -->
+            </div>
+            <!-- /.row -->
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- Kata -->
+
+    <a class="anchor" name="history"></a>
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <h2 class="section-heading">Historien til Eclipse Collections</h2>
+            <p class="lead">
+                Opprinnelig startet Eclipse Collections som et samlingsrammeverk kalt Caramel på Goldman Sachs i 2004. 
+                
+                Siden den gang har rammeverket utviklet seg, og i 2012 ble det lagt ut som åpen kiledekode på GitHub som et prosjekt kalt 
+                <a href="https://github.com/goldmansachs/gs-collections">GS Collections</a>.
+                <br><br>
+                GS Collections har blitt presentert på en rekke
+                <a href="https://github.com/eclipse/eclipse-collections/wiki/Conference-talks-and-meetups">konferanser</a>,
+                inkludert JVM Summit i 2012 og JavaOne i 2014.
+                En sammenligning av ytelse mellom parallelle lazy implementasjoner i Java 8, Scala og GS Collections ble presentert på
+                <a href="http://www.infoq.com/presentations/java-streams-scala-parallel-collections">QCon New York</a> i 2014.
+                Videre ble <a href="https://github.com/eclipse/eclipse-collections/wiki/Articles">artikler</a> om GS Collections
+                (<a href="http://www.infoq.com/articles/GS-Collections-by-Example-1">Del 1</a> /
+                <a href="http://www.infoq.com/articles/GS-Collections-by-Example-2">Del 2</a>)
+                publisert på InfoQ.com.
+                Disse artikler viser noen av funksjonene i rammeverket gjennom eksempler, samt et intervju med skaperen av GS Collections.
+                <br><br>
+
+                Gjennom årene har rundt 40 utviklere fra det samme selskapet bidratt til rammeverket.
+                <br><br>
+
+                For å maksimere prosjektets potensiale som et åpen kildekode-prosjekt, ble GS Collections migrert til 
```

Comment:
_Maksimere_ or _utnytte_? Or even _Fullt ut utnytte_?

Reply:
Yes! _"For å utnytte fullt ut prosjektets potensiale..."_.